src/components/filter/age-range-slider/age-range-slider.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { ChangeEvent } from "react";
+import { useFilterStore } from "../filter-store";
+import { useI18nStore } from "../../../i18n/i18n-store";
+
+export const INITIAL_MIN = 0;
+export const INITIAL_MAX = 200;
```

Comment:
right now the initial min/max values are declared in this file _and_ in the `filter-store.tsx`. I'd suggest to declare it only once (preferably in the filter-store) and import it from there, to have a single source of truth.